File: iqchannels-sdk/src/main/java/ru/iqchannels/sdk/schema/Response.java
package ru.iqchannels.sdk.schema;

import androidx.annotation.Nullable;

/**
 * Copyright iqstore.ru. All Rights Reserved.
 */
public class Response<T> {
    public boolean OK;
    @Nullable public ResponseError Error;
    @Nullable public T Result;
    @Nullable public Relations Rels;
}

File: iqchannels-sdk/src/main/java/ru/iqchannels/sdk/schema/Response.kt
package ru.iqchannels.sdk.schema

import com.google.gson.Gson

class Response<T> {
	var OK = false
	var Error: ResponseError? = null
	var Result: T? = null
	var Data: T? = null
	var Rels: Relations? = null

	override fun toString(): String {
		return Gson().toJson(this)
	}
}
